gulp/tasks/test.js

```diff
@@ -123,6 +124,30 @@ function _getWCAGParams() {
 }
 
 /**
+ * Processes command-line and environment variables
+ * for passing to the psi executable.
+ * An optional URL path comes from the command-line `--u=` flag.
+ * A PSI "strategy" (mobile vs desktop) can be specified with the `--s=` flag.
+ * @returns {Array} Array of command-line arguments for psi binary.
+ */
+function _getPSIParams( callback ) {
```

Comment:
Should this be named  `_createPSITunnel` since it's the main purpose of the function?